signature/policy_types.go

```diff
@@ -1,4 +1,5 @@
 // Note: Consider the API unstable until the code supports at least three different image formats or transports.
+//       Keep this in sync with documentation
```

Comment:
`policy_types.go` has nothing to do with `registries.d`. The relevant comment already is in https://github.com/containers/image/blob/master/docker/lookaside.go#L28 .